### PR TITLE
Grouper patch

### PR DESCRIPTION
**Pull request by ticoann**
replace #6459. @ericvaandering, Eric I add the dependency for rpm and JobArchiver change over your patch. Please review

### CODE REVIEW

File: src/python/WMComponent/JobArchiver/JobArchiverPoller.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
 import tarfile
 import traceback
 
+from Utils.IterTools import grouper
 from WMComponent.TaskArchiver.TaskArchiverPoller import uploadPublishWorkflow
 from WMCore.WorkerThreads.BaseWorkerThread import BaseWorkerThread
 from WMCore.JobStateMachine.ChangeState import ChangeState
```
Comment (amaltaro) [inline src/python/WMComponent/JobArchiver/JobArchiverPoller.py]: remove the import Lexicon please

Comment (ticoann) [inline src/python/WMComponent/JobArchiver/JobArchiverPoller.py]: Thanks Alan, it is removed now.

**Comment by ericvaandering** [comment]
So, if you do it right you can make a PR against another developer's branch. :-)

**Comment by ticoann** [comment]
> So, if you do it right you can make a PR against another developer's branch. :-)

Oh, I didn't think about that I will try next time.

**Pull request merged**